changelog/dmd.deprecation-version-debug-number.dd

```diff
@@ -0,0 +1,3 @@
+Integers in debug or version statements have been removed from the language
+
+These were deprecated in 2.101. Use `-debug=identifier` and `-version=identifier` instead.
```

Comment:
`Use `-debug=identifier` and `-version=identifier` instead for versions set on the command line, or `version = identifier;` and `debug = identifier;` for versions set in code at global scope.